CONTRIBUTING.md

```diff
@@ -253,17 +287,20 @@ cmake -DUNIT_TESTING=ON -DUNIT_TESTING_MOCK_ENABLED=ON ..
 
 See [compiler options section](#compiler-options) to learn about how to build samples with HTTP implementation in order to be runnable.
 
-After building samples with HTTP stack, set next environment variables to set log in credentials. Samples will read these values from env and use it to log in to Azure Service like Storage or KeyVault. Learn about the supported authentication [client secret here](https://docs.microsoft.com/en-us/azure/active-directory/azuread-dev/v1-oauth2-on-behalf-of-flow#service-to-service-access-token-request).
+After building samples with HTTP stack, set next environment variables for enabling credentials. Samples will read these values from env and use it to log in to Azure Service like Storage or KeyVault. Learn about the supported authentication [client secret here](https://docs.microsoft.com/en-us/azure/active-directory/azuread-dev/v1-oauth2-on-behalf-of-flow#service-to-service-access-token-request).
```

Comment:
```suggestion
After building samples with HTTP stack, set the environment variables for credentials.  The samples read these environment values to authenticate to Azure services.  See [client secret here](https://docs.microsoft.com/en-us/azure/active-directory/azuread-dev/v1-oauth2-on-behalf-of-flow#service-to-service-access-token-request) for additional details on Azure authentication.
```

Reply:
@gilbertw, I edited your suggestion to remove the extra white space after the period in `... to Azure services.  See [client ...` (assuming we use single white space after periods in most places). 

Please undo if I am mistaken.